src/v2/Components/ArtworkFilter/SavedSearch/Components/CreateAlertButton.tsx

```diff
@@ -27,6 +41,21 @@ export const CreateAlertButton: React.FC<CreateAlertButtonProps> = ({
     })
   }
 
+  const handleClick = () => {
+    if (isLoggedIn) {
+      handleOpenForm()
+    } else {
+      openAuthToFollowSaveCreate(mediator, {
+        entity: {
+          name: savedSearchAttributes.name,
+          slug: savedSearchAttributes.slug,
+        },
+        contextModule: ContextModule.artworkGrid,
+        intent: "createAlert" as AuthIntent,
```

Comment:
the intent for this action is not added to cohesion yet, left comment for @louislecluse in jira ticket about this

Reply:
Just created a PR for that here: https://github.com/artsy/cohesion/pull/274

Reply:
@dblandin thanks for prompt action!